moxie-vesting/src/mappings/tokenLockWallet.ts

```diff
@@ -90,6 +91,7 @@ export function handleTokenDestinationsRevoked(
 ): void {
   let tokenLockWallet = TokenLockWallet.load(event.address.toHexString())!
   tokenLockWallet.tokenDestinationsApproved = false
+  tokenLockWallet.tokenDestinationApprovalBlockNumber = BigInt.zero()
```

Comment:
implementation of `handleSubjectTokenDestinationsApproved` looks wrong. 
It should not track that in `tokenDestinationsApproved` variable.